build/buildall

```diff
@@ -246,3 +250,40 @@ time (
     -Dbuildver="$BASE_VER" \
     -DskipTests -Dskip -Dmaven.javadoc.skip
 )
+
+###############################################################################################
+# the following is to check the revisions in shims are identical
+###############################################################################################
+shims_for_check="${SPARK_SHIM_VERSIONS[@]}"
```

Comment:
I think it would be nice to put this in separate function, that way if one wants to comment it out its a single line to remove.

Reply:
updated